### PR TITLE
PWA-1630: Remove product from wish list fails if there are 2 or more products in a wishlist.

### PR DESCRIPTION
**Pull request by eug123**
## Description

URL - https://develop.pwa-venia.com

**Steps to reproduce existing bug** 
1. Go to venia as auth user
2. Add 2 products to favorite lists
3. Go to favorite lists and select Remove wishlist for a product.
4. Click Delete

**Expected** - User should be able to remove product from wishlist
**Actual** - User gets error, PFA screen.

### Fix
`value_id` field was missing in `ConfigurableWishlistItem` gql, also EE schema have more fields in `wishlist`, so using different wishlist schemas for CE and EE

## Related Issue

[[PWA-1630](https://jira.corp.magento.com/browse/PWA-1630)] Remove product from wish list fails if there are 2 or more products in a wishlist.

### Verification Steps

1. Add a simple product to wishlist and verify that you can delete it without seeing this error
2. Add a configurable product with no options selected to wishlist and verify that you can delete it without seeing this error
3. Add a configurable product with options selected to wishlist and verify that you can delete it without seeing this error
4. Add a simple product and configurable product with no options selected to wishlist and verify that you can delete it without seeing this error
5. Add two copies of the same configurable product - one with options selected, one without - to a wishlist and verify that you can delete them without seeing this error
6. Add two copies of the same configurable product - with two different option sets selected - to a wishlist and verify that you can delete it without seeing this error

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: packages/peregrine/lib/talons/WishlistPage/__tests__/useWishlistItem.spec.js [hunk 1]
```diff
@@ -85,3 +89,39 @@ test('handleAddToCart callback fires mutation', () => {
...
+    const talonProps = log.mock.calls[0][0];
+
+    act(() => {
+        talonProps.handleRemoveProductFromWishlist();
+    });
+
+    expect(mockMutate).toHaveBeenCalled();
+});
+
+test('handleRemoveProductFromWishlist callback error fires mutation', async () => {
```
Comment (revanth0212) [inline packages/peregrine/lib/talons/WishlistPage/__tests__/useWishlistItem.spec.js:107]:
```suggestion
test('handleRemoveProductFromWishlist callback logs error if the mutation fails', async () => {
```

**Review by revanth0212 — APPROVED** [review]
Nice work @eug123

A minor suggestion regarding the wording of a test. Other than that, the code looks ready to be merged.

**Comment by revanth0212** [comment]
@eug123 I have a bit of suggestion for you. It would be great if you can also provide in a sentence how you managed to fix the error in the description section.

**Comment by eug123** [comment]
@revanth0212, I have added explanation to description.
Thank you!

**Comment by dpatil-magento** [comment]
QA Approved

**Review by revanth0212 — APPROVED** [review]

**Pull request merged**